resources/domain/account/index.ts

```diff
@@ -2,8 +2,23 @@ import { getSignerDisplayType } from '../signer'
 
 export const accountNS = '114c39e5-cd7d-416f-ab9e-5ab6ab0218ce'
 
-export const isDefaultAccountName = ({ name, lastSignerType }: Account) =>
-  name.toLowerCase() === getDefaultAccountName(lastSignerType)
+export const isDefaultAccountName = ({ name, lastSignerType }: Account) => {
+  if (!lastSignerType) {
+    return false
+  }
+  const signerDisplayTypeMap = {
```

Comment:
it might just be easier to have the migration itself check for 'Address' or 'Watch' accounts, then we can put the code back to something a little more readable

Reply:
I was assuming that the migration shouldn't be touched because it is already in the codebase and it's not the latest one any more, though I guess we haven't had a prod release featuring this migration...

Reply:
if the migration already ran then there's no issue, it's only going to be a problem for users running this migration for the first time that their data will be entered into the meta incorrectly